### PR TITLE
Add readme for Prober and monitoring module.

### PR DESCRIPTION
**Pull request by sailorlqh**

### CODE REVIEW

**Review by sethvargo — APPROVED** [review]

File: docs/prober-and-monitoring.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Prober and Alerting
+
+**JVS is not an official Google product.**
+
+## Prober
+
+[Prober](../prober/) is a bash script that uses [jvsctl](../cmd/jvsctl/) to request and validate tokens. 
+
+In each prober job, the prober will request and validate a token with jvs service. The job will be treated as a success only if te following requirement is met.
```
Comment (yolocs) [inline docs/prober-and-monitoring.md:9]:
```suggestion
In each prober job, the prober will request and validate a token with jvs service. The job will be treated as a success only if the following requirement is met.
```

File: docs/prober-and-monitoring.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+- 1. A token is successfully requested.
+- 2. The same token is validated successfully.
```
Comment (yolocs) [inline docs/prober-and-monitoring.md:11]:
```suggestion
  1. Request a token successfully.
  2. Validate the token successfully.
```

File: docs/prober-and-monitoring.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+By default, prober will be deployed using [GCP's cloud run job](https://cloud.google.com/run/docs/create-jobs), and use [cloud scheduler](https://cloud.google.com/scheduler/docs/overview) to trigger the cloud run job on a user defined frequency.
```
Comment (yolocs) [inline docs/prober-and-monitoring.md:13]:
```suggestion
The prober will be deployed using [GCP's cloud run job](https://cloud.google.com/run/docs/create-jobs), and use [cloud scheduler](https://cloud.google.com/scheduler/docs/overview) to trigger the cloud run job on a user defined frequency.
```

File: docs/prober-and-monitoring.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+## Monitoring and Alert Policy
+
+We monitor JVS's UI-Service, API-Service, Cert-Rotator and Public-Key Service using metrics provided by [native cloud run monitoring metrics](https://cloud.google.com/monitoring/api/metrics_gcp#gcp-run), the selected metrics are:
```
Comment (yolocs) [inline docs/prober-and-monitoring.md:17]:
```suggestion
We monitor all JVS services with [native cloud run monitoring metrics](https://cloud.google.com/monitoring/api/metrics_gcp#gcp-run) and alert based on the following metrics:
```

File: docs/prober-and-monitoring.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+-  Request Count 
+-  Request Latency
+
+And we also create alert policy for each JVS service on the above two metrics.
+
+We also created alert policies base on cloud run job result for prober service. The default policy is: In a sliding window of 60 minutes, if the number of failed prober cloud run job exceeds 4, a alert will be sent.
```
Comment (yolocs) [inline docs/prober-and-monitoring.md:24]:
```suggestion
Similarly, we alert base on cloud run job execution metrics for prober service.

See alert policies in FILE(link to terraform file).
```

We don't want to duplicate documentation we already have in code.

File: docs/prober-and-monitoring.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+  notification_channel_email     = "Email to which alert will be sent to"
+  prober_jvs_api_address         = "https://jvs.corp.internal:8080"
+  prober_jvs_public_key_endpoint = "https://keys.corp.internal:8080/.well-known/jwks"
+  jvs_prober_image               = "us-docker.pkg.dev/abcxyz-artifacts/docker-images/jvs-prober:0.0.5-amd64"
+  prober_audience                = "This would be either your gcloud app id, or jvs-api service's url"
+}
+```
+
+By default, alering is disabled, you are enable it by setting the following variables:
+```
```
Comment (yolocs) [inline docs/prober-and-monitoring.md:49]:
```suggestion
By default, alering is disabled, you are enable it by setting the following variables:

```
```

File: docs/prober-and-monitoring.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+alert_enabled        = true
+prober_alert_enabled = true
```
Comment (yolocs) [inline docs/prober-and-monitoring.md:51]: It seems we can combine these flags in a follow-up?

**Review by yolocs — CHANGES_REQUESTED** [review]

**Review by yolocs — APPROVED** [review]

**Pull request merged**